core/src/load_library.rs

```diff
@@ -88,8 +82,8 @@ pub fn load_mongosqltranslate_library() {
     });
 }
 
-pub fn get_run_command() -> Result<
-    Symbol<'static, unsafe extern "C" fn(*const u8, usize) -> RunCommandResult>,
+pub fn get_mock_run_command() -> Result<
```

Comment:
I changed it because it can only be used for the mock runCommand due to this part: `unsafe extern "C" fn(*const u8, usize)`. The other one requires `unsafe extern "C" fn(BsonBuffer)`.